packages/core/src/lib/GradleWrapper.ts

```diff
@@ -62,4 +62,14 @@ export class GradleWrapper {
     await executeFile(
         this.gradleCmd, ['assembleRelease', '--stacktrace'], env, undefined, this.projectLocation);
   }
+
+  /**
+   * Executes gradle commands with custom arguments.
+   * @param args - Arguments supplied to gradle, also considered gradle tasks.
+   */
+  async executeGradleCommand(args: string[]): Promise<void> {
```

Comment:
We can now implement the above two methods (maybe more as well) just using this one.

Or maybe we wanted to keep all the raw gradle commands in this file (to make them easier to audit maybe), in which case we should make a `bootstrap` method or something.

Reply:
> We can now implement the above two methods (maybe more as well) just using this one.

Yup, that was my intention moving forward!

> Or maybe we wanted to keep all the raw gradle commands in this file (to make them easier to audit maybe), in which case we should make a bootstrap method or something.

I was torn between putting the bootstrap command in here or putting in its own Play.ts file. The reasoning behind the Play.ts file was that if we do decide to move away from the plugin in the future, it may make the API surface a bit more stable if we still have the Play.ts file hanging around that we can call. Might also make it a little more stable if we just make a Play interface with which the GradleWrapper can implement the methods and if we do decide to move away from the Gradle plugin, we can just implement the interface on another class?

Thoughts?